packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -226,6 +228,7 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
             <ThreadCard
               key={thread.id + '-' + thread.readOnly}
               thread={thread}
+              canReact={!(!hasJoinedCommunity || !!thread.lockedAt || !!thread.archivedAt)}
```

Comment:
Can we make this logic more readable, maybe by assigning it to a variable first?

Reply:
Done